src/Api/Projects.php

```diff
@@ -129,10 +139,6 @@ public function all(array $parameters = [])
             ->setNormalizer('repository_checksum_failed', $booleanNormalizer)
         ;
         $resolver->setDefined('repository_storage');
-        $resolver->setDefined('starred')
```

Comment:
?

Reply:
This was accidentally added: the 'starred' parameter was already supported.